src/main/java/step2/ui/InputView.java

```diff
@@ -1,50 +1,63 @@
 
 public class InputView {
 
+    public static final String PURCHASE_AMOUNT_MSG = "구입금액을 입력해 주세요.";
+    public static final String BONUS_NUMBER_MSG = "보너스 볼 번호를 입력해 주세요.";
+    public static final String PREVIOUS_NUMBER_MSG = "지난 주 당첨 번호를 입력해 주세요.";
+    public static final String MANUAL_GAMES_COUNT_MSG = "수동으로 구매할 로또 수를 입력해 주세요";
+    public static final String MANUAL_GAMES_MSG = "수동으로 구매할 번호를 입력해 주세요.";
     public static final int AMOUNT_OF_UNITS = 1000;
 
-    public static void startMsg() {
-        System.out.println("구입금액을 입력해 주세요.");
+    public static int getPurchaseAmount() {
+        printAnnounceMsg(PURCHASE_AMOUNT_MSG);
+        return validateProofOfAmount(getInputNumber()) / AMOUNT_OF_UNITS;
     }
 
-    public static int moneyInput() {
+    public static String getInputString() {
         Scanner scanner = new Scanner(System.in);
-        int money = scanner.nextInt();
-        if (proofOfAmount(money)) {
-            throw new IllegalArgumentException("금액이 잘못 되었습니다.");
-        }
-        return money / AMOUNT_OF_UNITS;
+        return scanner.nextLine();
     }
 
-    static boolean proofOfAmount(int money) {
-        return money > 1000
-                && money % 1000 != 0;
+    public static int getInputNumber() {
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextInt();
     }
 
-    public static void printTotalGames(int totalGames) {
-        System.out.println(totalGames + "개를 구매했습니다.");
+    public static void printAnnounceMsg(String announcementMessage) {
+        System.out.println(announcementMessage);
     }
 
-    public static void enterThePreviousNumberMsg() {
-        System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+    public static String getPreviousNumber() {
+        printAnnounceMsg(PREVIOUS_NUMBER_MSG);
+        return validateFormula(getInputString());
     }
 
-    public static void enterTheBonusNumberNumberMsg() {
-        System.out.println("보너스 볼 번호를 입력해 주세요.");
+    public static int getBonusNumber() {
+        printAnnounceMsg(BONUS_NUMBER_MSG);
+        return validateLottoRangeNumber(getInputNumber());
     }
 
-    public static String getPreviousNumber() {
-        enterThePreviousNumberMsg();
-        Scanner scanner = new Scanner(System.in);
-        return scanner.nextLine();
+    public static int getTotalManualGameCount() {
+        printAnnounceMsg(MANUAL_GAMES_COUNT_MSG);
+        return validateLottoRangeNumber(getInputNumber());
     }
 
-    public static int getBonusNumber() {
-        enterTheBonusNumberNumberMsg();
-        Scanner scanner = new Scanner(System.in);
-        return scanner.nextInt();
+    public static List<String> getManualGame(int totalManualGameCount) {
+        printAnnounceMsg(MANUAL_GAMES_MSG);
+        return Arrays.asList()
+                     .stream()
+                     .map(s -> getInputString())
+                     .collect(Collectors.toCollection(() -> new ArrayList<>(totalManualGameCount)));
```

Comment:
Arrays.asList()를 통해 빈 List에 대해 Stream 연산을 수행하는걸로 보이는데

empty Stream이라 원하는 getInputString()이 실행되지 않아 로또 수동은 아무리 해도 입력되지 않을 것 같아요.

원하시는걸 수행하기 위해선 아래와 같이 정의되어야 할 것 으로 보입니다.

```suggestion
return Stream.generate(InputView::getInputString)
                .limit(totalManualGameCount)
                .collect(Collectors.toList());
```

배포 및 구동이 어려운 상황이 아니니, 로컬에서 간단하게 실행시켜 메뉴얼로 확인하고 올려주시면 좋을 것 같아요.

Reply:
감사합니다.
개발 하면서 조금 착각을 했던 것 같습니다.!